packages/cozy-client/src/associations/Association.js

```diff
@@ -223,6 +223,10 @@ class Association {
   static query() {
     throw new Error('A custom relationship must define its query() function')
   }
+
+  dehydrate(doc) {
+    return doc
+  }
```

Comment:
Là comme ça ? Ça a pété une erreur sans ?

Reply:
This is removed later on. I had quite some trouble to have optional methods on the Association class.